### PR TITLE
Add netMHCIIpan 4.1 interface

### LINKED ISSUE #65: Include NetMHCIIpan 4.1 interface
There is a new class 2 predictor NetMHCIIpan 4.1, which would be interesting to squeeze in before the release

### PR DESCRIPTION
**Pull request by jonasscheid**
This adds an interface for netMHCIIpan 4.1. 

Supported alleles are identical to 4.0, which is already implemented

Fixes #65

### CODE REVIEW

File: epytope/EpitopePrediction/External.py [hunk 1]
```diff
@@ -8460,7 +8510,8 @@ class ScoreIndex(IntEnum):
     NETMHCII_2_3 = 5
     NETMHCIIPAN_3_0 = 3
     NETMHCIIPAN_3_1 = 3
-    NETMHCIIPAN_4_0 = 3
+    NETMHCIIPAN_4_0 = 4
```
Comment (jonasscheid) [inline epytope/EpitopePrediction/External.py:8513]: I noticed that the enums were wrong here

File: epytope/EpitopePrediction/External.py [hunk 2]
```diff
@@ -8492,7 +8544,8 @@ class Offset(IntEnum):
     NETMHCSTABPAN_1_0_WO_SCORE = 3
     NETMHCIIPAN_3_0 = 3
     NETMHCIIPAN_3_1 = 3
-    NETMHCIIPAN_4_0 = 1
+    NETMHCIIPAN_4_0 = 0
```
Comment (jonasscheid) [inline epytope/EpitopePrediction/External.py:8547]: I noticed that the enums were wrong here

**Review by christopher-mohr — APPROVED** [review]
LGTM 👍, you could add it to the `CHANGELOG`.

**Comment by jonasscheid** [comment]
👍 Will add it in #64 once this one is merged

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Added`
 
 - [#58](https://github.com/KohlbacherLab/epytope/pull/58) - Add check for `BioMart` transcript sequence availability [#55](https://github.com/KohlbacherLab/epytope/issues/55)
-- [#59](https://github.com/KohlbacherLab/epytope/pull/59) - Add interface for `NetMHCpan 4.1`
+- [#59](https://github.com/KohlbacherLab/epytope/pull/59) - Add interface for `NetMHCpan 4.1` [#56](https://github.com/KohlbacherLab/epytope/issues/56)
+- [#66](https://github.com/KohlbacherLab/epytope/pull/59) - Add interface for `NetMHCIIpan 4.1` [#65](https://github.com/KohlbacherLab/epytope/issues/65)
```
Comment (christopher-mohr) [inline CHANGELOG.md:12]:
```suggestion
- [#66](https://github.com/KohlbacherLab/epytope/pull/66) - Add interface for `NetMHCIIpan 4.1` [#65](https://github.com/KohlbacherLab/epytope/issues/65)
```

**Review by christopher-mohr — CHANGES_REQUESTED** [review]
One small change

**Review by christopher-mohr — APPROVED** [review]

**Pull request merged**